src/test/java/org/eclipse/basyx/testsuite/regression/extensions/submodel/aggregator/mqtt/TestMqttV2SubmodelAggregatorObserver.java

```diff
@@ -120,18 +124,25 @@ public void tearDown() {
 
 	@Test
 	public void testCreateSubmodel() {
-		String newSubmodelIdShort = "newSubmodelIdShort";
-		String newSubmodelId = "newSubmodelId";
-		Identifier newSubmodelIdentifier = new Identifier(IdentifierType.IRDI, newSubmodelId);
-		Submodel newSubmodel = new Submodel(newSubmodelIdShort, newSubmodelIdentifier);
-		newSubmodel.addSubmodelElement(new SubmodelElementCollection("smeColl"));
-		newSubmodel.addSubmodelElement(new Property("prop", false));
+		Submodel newSubmodel = createTestSubmodel();
 		observedSubmodelAggregator.createSubmodel(newSubmodel);
 
 		assertEquals(removeSubmodelElements(newSubmodel), deserializePayload(listener.lastPayload));
 		assertEquals(payloadFactory.createCreateSubmodelTopic(null, observedSubmodelAggregator.getAasServerId()), listener.lastTopic);
 	}
 
+	@Test
+	public void testCreateSubmodelViaApi() {
```

Comment:
It would be better to rename this test method such that the below comment is not required, for e.g., noEventExpectedWhenSubmodelIsCreatedViaAPI